esphome/components/opentherm/switch/switch.cpp

```diff
@@ -0,0 +1,27 @@
+#include "switch.h"
+
+namespace esphome {
+namespace opentherm {
+
+void OpenthermSwitch::write_state(bool state) { this->publish_state(state); }
+
+void OpenthermSwitch::setup() {
+  auto restored = this->get_initial_state_with_restore_mode();
+  bool state = false;
+  if (!restored.has_value()) {
+    ESP_LOGD("opentherm.switch", "Couldn't restore state for OpenTherm switch '%s'", this->get_name().c_str());
```

Comment:
Please update to add TAG to all the cpp files and use that for logging
```suggestion
    ESP_LOGD(TAG, "Couldn't restore state for OpenTherm switch '%s'", this->get_name().c_str());
```